tools/travis/stage.sh

```diff
@@ -164,6 +168,9 @@ function get_compile_modules_for_stage() {
         (${STAGE_TESTS})
             echo "-pl $MODULES_TESTS -am"
         ;;
+        (${STAGE_CONNECTOR_HIVE_1})
+            echo "-pl $MODULES_CONNECTOR_HIVE -am -Phive-1.2.1 clean"
```

Comment:
My guess is you're using clean so it is recompiled with a different hive version?

Reply:
Yes